package.json

```diff
@@ -44,8 +44,8 @@
     "jasmine-node": "^1.14.5",
     "jest": "^17.0.3",
     "react": "^15.0.0",
-    "react-addons-test-utils": "^15.4.2",
-    "react-dom": "^15.0.0",
+    "react-dom": "^15.5.4",
```

Comment:
@developerdizzle if you leave it at "^15.0.0" then "react-dom" starts throwing the same prop-types errors that we saw in #54. To suppress warnings, and get green all across the board i would recommend it :)